framework/src/modules/chain/bft/bft.js

```diff
@@ -182,48 +177,6 @@ class BFT extends EventEmitter {
 		);
 	}
 
-	/**
-	 * Computes maxHeightPreviouslyForged and maxHeightPrevoted properties that are necessary
-	 * for creating a new block
-	 * @param delegatePublicKey
-	 * @return {Promise<{maxHeightPrevoted: number, maxHeightPreviouslyForged: (number|*)}>}
-	 */
-	async computeBFTHeaderProperties(delegatePublicKey) {
```

Comment:
Where'd this logic go?

Reply:
It's in the processor v2.
maxHeightPreviouslyForged is calculated in v2, and maxHeightPrevoted is getting from getter

Reply:
I'm not sure about moving this logic to block processor v2, what's the reason behind it? Feels like it belongs to `bft.js` or at least `bft` folder.

Reply:
maxForgedHeight is related to `BFT`, but managed by forging module.
it's out of `BFT` because `forger_info` module is nothing to do with `bft`, and we will include `dpos` related information in the `forger_info` in the future too